test/app.integration.spec.js

```diff
@@ -302,97 +280,85 @@ describe('Test purchasesOrders:', () => {
         done();
       });
   });
+  it('GET / Id purchasesOrders : bad ID', (done) => {
+    request(app)
+      .get('/purchasesOrders/noId')
+      .expect(404)
+      .expect('Content-Type', /json/)
+      .then(response => {
+        const expected = { message: "PurchasesOrders ID not found" }
+        expect(response.body).toEqual(expected);
+        done();
+      });
+  });
+  it('GET / Id purchasesOrders : ID not found', (done) => {
+    request(app)
+      .get('/purchasesOrders/2000')
+      .expect(404)
+      .expect('Content-Type', /json/)
+      .then(response => {
+        const expected = { message: 'PurchasesOrders ID not found' }
+        expect(response.body).toEqual(expected);
+        done();
+      });
+  });
+  it('GET / Id purchasesOrders : Correct', (done) => {
+    request(app)
+      .get('/purchasesOrders/1')
+      .expect(200)
+      .expect('Content-Type', /json/)
+      .then(response => {
+        const expected = { DVM_id: expect.any(Number), id: expect.any(Number), quantity: expect.any(Number), }
+        expect(response.body[0]).toEqual(expected)
```

Comment:
GET /purchasesOrders/1 shouldn't response with an array, so the test should check response.body, not response.body[0]